supervisor/testing_analyst.py

```diff
@@ -59,13 +64,18 @@ def render_prompt(input: InputSchema) -> str:
       If the tests are currently running:
          state: tests-running
          comment: [Provide a brief description of what tests are running and where the results will be]
+
+      If the erratum's status is QE and there are no comments from NEWA stating it has initiated test execution within 6 hours after the erratum's last status transition timestamp:
```

Comment:
This is specific to only some components - not even all components that use NEWA - so let's put it into the QE information database instead of hardcoding here. Any text here should be more generic.

s/If the tests need to be started manually/If the tests need to be started manually or if they are supposed to be started manually but too much time has passed/ - something like that.